DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+In the LSST stack \citep{LDM-151} framework, each reduced input image
+is call a ``calexp'' (Calibrated Exposure). Each calexp holds the
+data from one exposure of one CCD, and we associate the ``reduction''
+products, typically a variance map, image mask planes, and the derived
+catalog and WCS obtained by matching the catalog to some external
+reference during single-frame processing. The data that jointcal needs
+from the calexp is stored in a \ClName{CcdImage} object. It stores
+the objects selected from the source catalog (using a configurable
+\ClName{SourceSelector}), the relevant exposure metadata and the
+initial WCS and \ClName{PhotoCalib}.
+
+\begin{figure}[ht]
+\centering{}\includegraphics[width=0.8\textwidth]{_static/AssocClasses}
+\caption{Chart of class relations which implement the associations between
+input catalogs. One \ClName{FittedStar} usually has several \ClName{MeasuredStar}
+pointing to it, and each \ClName{RefStar} points to exactly one
+\ClName{FittedStar}. Most \ClName{FittedStar's} have no \ClName{RefStar}.
+\label{fig:AssocClasses} }
+\end{figure}
+
+The \ClName{Associations} class holds the list of input \ClName{CcdImage's}
+and connects together the measurements of the same object. The input
+measurements are called \ClName{MeasuredStar} and the common detections
+are called \ClName{FittedStar}. The objects collected in an external
+catalog are called \ClName{RefStar}. Despite their names, these
+classes can represent galaxies as well as stars. The collections of
+sush objects are stored into \ClName{MeasuredStarList}, \ClName{RefStarList}
+and \ClName{FittedStarList}, which are container derived from \ClName{std::list}.
+The relations between these classes, all implemented in C++, are displayed
+in figure \ref{fig:AssocClasses}.
+
+\section{Fitting the transformations between a set of images}
+
+Some applications require determination of transformations between
+images rather than mappings on the sky. For example a simultaneous
+fit of PSF photometry for the computation of the light curve a point-like
+transient requires mappings between images to transport the common
+position in pixel space from some reference image to any other in
+the series. The calexp series would typically involve the CCD from
+each exposure that covers the region of interest. The package described
+here can fit for the needed mappings: 
+\begin{itemize}
+\item in order to remove all reference terms from the $\chi^{2}$ of eq.~\ref{eq:astrometry-chi2},
+one avoids calling \RoutineName{Associations::CollectRefStars}. 
+\item One chooses polynomial mappings for all Calexp except, reserving one
+to serve as a reference with a fixed identity mapping. The distortion
+model \ClName{SimplePolyModel} allows that. 
+\item Chose identity projectors (the class \ClName{IdentityProjector}
+does precisely that). 
+\end{itemize}
+So, fitting transformations between image sets can be done with the
+provided code.
+
+\section{Run example}
+\begin{verbatim}
```

Comment:
Use the `{lstlisting}` environment here.

Reply:
I'm removing that whole section, since there's a `jointcalTask` now.